style.css

```diff
@@ -1,18 +1,763 @@
+.description__link_active {
+border: 2px solid #FFFFFF;
+width: 135px;
+height: 15px;
+}
+
+.parameters__circle{
+margin-right: 10px;
+}
+
+
+.safety{
+display: flex;
+flex-direction: row;
+position: relative;
+}
+
+.safety__block{
+display: flex;
+position: relative;
+flex-direction: column;
+align-items: center;
+width: 30%;
+height: 100%;
+background-color: #E51937;
+color: #FFFFFF;
+}
+
+.safety__description{
+width: 316px;
+margin-top: 20px;
+}
+
+.safety__title{
+font-style: normal;
+font-weight: 600;
+font-size: 30px;
+line-height: 37px;
+}
+
+.safety__text{
+font-style: normal;
+font-weight: 500;
+font-size: 15px;
+line-height: 18px;
+}
+
+.stars{
+position: relative;
+max-width: 316px;
+}
+
+.stars__subtitle{
+font-weight: 600;
+font-size: 30px;
+line-height: 37px;
+}
+
+.stars__list{
+display: flex;
+position: relative;
+width: 100%;
+margin-bottom: 10px;
+justify-content: space-between;
+}
+
+.stars__item{
+position: relative;
+font-style: normal;
+font-weight: bold;
+font-size: 17px;
+line-height: 21px;
+text-transform: uppercase;
+}
+
+.stars__block{
+margin-right: 30px;
+}
+
+.stars__spansmall{
+font-style: normal;
+font-weight: bold;
+font-size: 10px;
+line-height: 12px;
+text-transform: uppercase;
+opacity: 0.7;
+}
+
+.fivestars{
+display: flex;
+flex-direction: column;
+width: 347px;
+height: 250px;
+}
+
+.fivestars__safety{
+display: flex;
+flex-direction: row;
+margin-top: 80px;
+}
+
+.fivestars__shield{
+width: 62px;
+height: 74px;
+}
+
+.fivestars__textstars{
+position: relative;
+margin-left: 10px;
+margin-top: 0;
+margin-bottom: 0;
+font-style: normal;
+font-weight: 600;
+font-size: 50px;
+line-height: 30px;
+}
+
+.fivestars__span{
+font-style: normal;
+font-weight: 600;
+font-size: 25px;
+}
+
+.fivestars__active{
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+margin-top: 30px;
+}
+
+.fivestars__menu{
+display: flex;
+font-style: normal;
+font-weight: bold;
+font-size: 13px;
+line-height: 16px;
+text-align: center;
+text-transform: uppercase;
+color: #FFFFFF;
+text-decoration: none;
+}
+
+.fivestars__menu_active{
+border: 2px solid #FFFFFF;
+width: 160px;
+height: 40px;
+display: flex;
+align-items: center;
+justify-content: center;
+margin-right: 10px;
+}
+
+.fivestars__link{
+display: flex;
+align-items: center;
+font-style: normal;
+font-weight: bold;
+font-size: 13px;
+line-height: 16px;
+text-align: center;
+text-transform: uppercase;
+color: #FFFFFF;
+text-decoration: none;
+}
+
+.fivestars__circle{
+margin-right: 10px;
+}
+
+.schema{
+position: relative;
+width: 70%;
+height: 100%;
+display: flex;
+flex-direction: column;
+}
+
+.schema__title{
+position: relative;
+display: flex;
+width: 80%;
+top: 100px;
+left: 17%;
+}
+
+.schema__imgcar{
+display: flex;
+position: relative;
+width: 100%;
+}
+
+
+
+@media (max-width:1440px){
+	body{
+	max-width: 1400px;
+	}
+
+	.header{
+		max-width: 1400px;
+	}
+	.tesla{
+		max-width: 1400px;
+	}
+}
+
+@media (max-width:992px){
+	body{
+	max-width: 768px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)